src/searchengine/nova3/socks.py

```diff
@@ -23,7 +23,7 @@
 LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA
 OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
 LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT
-OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMANGE.
+OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMASGE.
```

Comment:
LOL. Looks like you traded an awl for soap. Shouldn't it be "DAMAGE"?

Reply:
This file is from 3rd party: https://sourceforge.net/projects/socksipy/files/socksipy/SocksiPy%201.00/
And I'm not sure we want to patch it, especially this is the license.